src/System.Private.ServiceModel/src/System/IdentityModel/Claims/X509CertificateClaimSet.cs

```diff
@@ -339,6 +375,59 @@ public IIdentity Identity
                 get { return _identity; }
             }
         }
+
+        // We don't have a strongly typed extension to parse Subject Alt Names, so we have to do a workaround 
+        // to figure out what the identifier, delimiter, and separator is by using a well-known extension
+        private static class X509SubjectAlternativeNameConstants
+        {
+            public const string Oid = "2.5.29.17"; 
+            public static readonly string Identifier;
+            public static readonly char Delimiter;
+            public static readonly string Separator;
+
+            // static initializer runs only when one of the properties is accessed
+            static X509SubjectAlternativeNameConstants()
+            {
```

Comment:
Good point. In order not to muck up the ordering of the commits here I'll merge this and immediately open up another PR to fix this issue.